comanage_utils.py

```diff
@@ -81,22 +81,30 @@ def call_api2(method, target, endpoint, authstr, **kw):
 
 def call_api3(method, target, data, endpoint, authstr, **kw):
     req = mkrequest(method, target, data, endpoint, authstr, **kw)
-    trying = True
-    currentTimeout = MIN_TIMEOUT
-    while trying:
+    retries = 0
```

Comment:
```suggestion
    retries = 1
```

Otherwise we actually try 6 times. Not 0 indexing feels kinda bad but I think it's better than setting `MAX_RETRIES = 4`

Reply:
I'll change where the retries gets incremented and the `while retries <= MAX_RETRIES` to `while retries < MAX_RETRIES` so that the values of retries will be 0, 1, 2, 3, 4 and then throw the exception. `MAX_RETRIES` should also be changed to `MAX_TRIES` to more accurately reflect what it represents, same with `retries` to something like `tries` or `req_attempts`.

Reply:
I've changed `retries` to `attempts`, moved around where we increment it, and changed `while retries <= MAX_RETRIES:` to `while req_attempts < MAX_ATTEMPTS:` so that we can start at 0 and still have MAX_ATTEMPTS total attempts.